src/layouts/ResourceRoom/ResourceRoom.tsx

```diff
@@ -82,40 +86,72 @@ const EmptyResourceRoom = () => {
     }
   }, [errorToast, isError])
 
+  const { isOpen, onOpen, onClose } = useDisclosure()
+
   return (
     <SiteViewLayout>
-      <Box as="form" w="full">
+      <Box as="form" w="full" mt={100}>
         {/* Resource Room does not exist */}
-        <Section align="flex-start">
-          <SectionHeader label="Create Resource Room" />
-          <SectionCaption label="NOTE: " icon={BiInfoCircle}>
-            You must create a Resource Room before you can create Resources.
-          </SectionCaption>
-          {/* Info segment */}
-          <FormControl
-            isRequired
-            isInvalid={!!errors.newDirectoryName?.message}
-          >
-            <Input
-              maxW="50%"
-              {...register("newDirectoryName", { required: true })}
-            />
-            <FormErrorMessage>
-              {errors.newDirectoryName?.message}
-            </FormErrorMessage>
-          </FormControl>
-          {/* Segment divider  */}
-          <Divider color="border.divider.alt" />
+        <VStack spacing={5}>
+          <EmptyBoxImage />
+          <VStack spacing={0}>
+            <Center textStyle="subhead-1">
+              There&apos;s nothing here yet.
+            </Center>
+            <Center textStyle="body-2">
+              Create a resource room to get started.
+            </Center>
+          </VStack>
           <Button
-            type="submit"
-            isLoading={isLoading}
-            onClick={handleSubmit(onSubmit)}
-            isDisabled={!_.isEmpty(errors)}
+            variant="solid"
+            onClick={onOpen}
+            leftIcon={<Icon as={BiPlus} fontSize="1.5rem" fill="white" />}
           >
             Create Resource Room
           </Button>
-        </Section>
+        </VStack>
       </Box>
+
+      <Modal blockScrollOnMount={false} isOpen={isOpen} onClose={onClose}>
+        <ModalOverlay />
+        <ModalContent>
+          <ModalHeader>Create resource room</ModalHeader>
+          <ModalCloseButton />
+          <ModalBody>
+            <Text textStyle="subhead-1">Resource room title</Text>
+
+            <FormControl
+              isRequired
+              isInvalid={!!errors.newDirectoryName?.message}
+            >
+              <Input
+                marginTop={5}
+                placeholder="Resource room name"
+                {...register("newDirectoryName", {
+                  required: "Please enter resource room name",
+                })}
+              />
```

Comment:
@seaerchin
The eslint gives and error saying: "Prop spreading is forbidden"

However, I do see this in other areas of the codebase.. Are we ignoring this warning as a team for now?

Reply:
Hmm, this should be a warning and not an error; I'm in favour of ignoring this as this is a design pattern given by the `react-hook-form` docs and we shouldn't need to care about the individual props returned